src/main/java/frc/trigon/robot/subsystems/arm/ArmIO.java

```diff
@@ -0,0 +1,57 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix6.StatusSignal;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.arm.toohardarm.ToohardArmIO;
+import org.littletonrobotics.junction.AutoLog;
+
+public class ArmIO {
+
+    protected void updateInputs(ArmInputsAutoLogged inputs) {
+
+    }
+
+    protected void setAngleMotorPower(double power) {
+
+    }
+
+    protected void setElevatorMotorPower(double power) {
+
+    }
+
+    protected void stopAngleMotor() {
+
+    }
+
+    protected void stopElevatorMotor() {
+
+    }
+
+    @AutoLog
+    protected static class ArmInputs {
+        public double
+                masterAngleMotorVoltage = 0,
+                followerAngleMotorVoltage = 0,
+                masterElevatorMotorVoltage = 0,
+                followerElevatorMotorVoltage = 0;
```

Comment:
No need to pass for both follower and master
just pass for `angle` and `elevator`